base/field_abilities/PoundFieldPsynergy.ts

```diff
@@ -0,0 +1,266 @@
+                    event.deactivate();
+                    event.is_set = false;
+                } else {
+                    event.activate();
+                    event.is_set = true;
+                    JumpEvent.active_jump_surroundings(
+                        this.data,
+                        get_surroundings(event.x, event.y, false, 2),
+                        event.collision_layer_shift_from_source + this.target_object.base_collision_layer
+                    );
+                }
+            });
+            let shadow_index = this.data.npc_group.getChildIndex(this.data.hero.sprite) - 1;
+            this.data.npc_group.setChildIndex(this.target_object.sprite, shadow_index);
+            this.target_object.sprite.send_to_back = true;
+            this.data.npc_group.sendToBack(this.target_object.sprite);
+            this.target_object.sprite.body.removeCollisionGroup(this.data.collision.hero_collision_group, true);
+        }
+        this.camera_shake(-8, 8);
+        this.init_stars();
+        this.init_wave(this.hand_sprite.centerX, this.hand_sprite.centerY);
+        this.hand_translate_x = this.hand_sprite.centerX;
+        this.hand_translate_y = this.controllable_char.sprite.centerY;
+        this.move_hand_by_dir(0, -12);
+        let timer = this.game.time.create(true);
+        timer.add(800, () => {
+            this.data.audio.play_se("pound2/se1");
+            this.hand_tween(this.hand_translate_x, this.hand_translate_y, 350, () => {
+                this.game.add.tween(this.hand_sprite).to({width: 0, height: 0}, 350, Phaser.Easing.Linear.None, true);
+                this.hand_translate_x = this.controllable_char.sprite.centerX;
+                this.hand_translate_y = this.controllable_char.sprite.centerY;
+                this.hand_tween(this.hand_translate_x, this.hand_translate_y, 400, () => {
+                    this.unset_hero_cast_anim();
+                    this.stop_casting();
+                    this.data.overlayer_group.remove(this.hand_sprite, false);
+                });
+            });
+        });
+        timer.start();
+    }
+
+    //Change hand movement depending on the cast direction
+    move_hand_by_dir(final_x, final_y) {
+        switch (this.cast_direction) {
+            case directions.up:
+                final_y -= 12;
+                final_x *= 0;
+                break;
+            case directions.down:
+                final_y += 4;
+                final_x *= 0;
+                break;
+            case directions.left:
+                final_x *= -1;
+                break;
+        }
+        this.hand_translate_x += final_x;
+        this.hand_translate_y += final_y;
+    }
+
+    hand_tween(var_x, var_y, time, complete_action) {
+        this.game.add
+            .tween(this.hand_sprite)
+            .to({centerX: var_x, centerY: var_y}, time, Phaser.Easing.Linear.None, true)
+            .onComplete.addOnce(complete_action);
+    }
+
+    init_stars() {
```

Comment:
The stars animation is not quite accurate, they have a bigger lifespan and a little bit of randomness on their positioning.